src/Microsoft.Azure.SignalR.Protocols/SequenceOfT.cs

```diff
@@ -0,0 +1,465 @@
+﻿// Copyright (c) Andrew Arnott. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.

```

Comment:
You'll either need to modify the LICENSE.txt reference to where you can *actually* find the license information, or update LICENSE.txt in the root of this repo to include the license info from the repo that you got this from.

Reply:
@vicancy, it's your call here

Reply:
`LICENSE.txt` to `LICENSE`? (https://github.com/Azure/azure-signalr/blob/dev/LICENSE)

Reply:
And I guess we need to merge in a copyright/license from the source repository (Nerdbank.Streams in this case) or put this in 3rdpartynotices.txt or something.

Reply:
@AArnott, can you check the 2 last commits to be sure this is what you meant ?
@vicancy is these 2 ok for you ?

I understand there is a duplicate of information, is it too much or not ?

Reply:
I created a small PR here https://github.com/AArnott/Nerdbank.Streams/pull/159
as `NerdBank.Streams` contains `LICENSE` and not `LICENSE.txt` just in case someone wonder about that